src/nav/accounts.yml

```diff
@@ -73,7 +73,24 @@ pages:
           - title: Videos of user management UI
             path: /docs/accounts/accounts-billing/new-relic-one-user-management/user-mgmt-videos            
           - title: Intro to SAML SSO and SCIM
```

Comment:
@zuluecho9, I found this intro doc a bit confusing as a solitary node on the same level as "Automated user management." You might consider just moving it to the first node under "Automated user management."

Reply:
it's a bit broader than that actually, because SAML SSO is separate from SCIM. So it's meant to serve as an overview of our offerings for both SAML SSO and SCIM, which are kind of related.